### DISCUSSION TITLE
Smart Playlist Generation - Use Absolute Path instead of Relative

### DISCUSSION THREAD

**Discussion by MickMorley** [jcorporation/myMPD#1211] [Q&A] [answered]
In my use case, I leverage (several) MyMpd containers as my manager for my (several) MPD containers. 

However, I also have a Jellyfin container to play my music on local workstations or mobile devices.  Jellyfin requires the paths in the playlists to be absolute.  When MyMpd generates or updates a SmartPlaylist (or any playlist) it uses the relative path to the media folder.

Is there a way now or is it possible to add a setting in MyMpd to use the absolute path when creating/updating playlists?

For example

MPD conf
`music_directory		"/media/music"`

Playlist (m3u) created by MyMPD
```
Apocalyptica/7th Symphony (2010)/07 - Broken Pieces.mp3
Breaking Benjamin/(2006) Phobia/01 Intro.mp3
Breaking Benjamin/(2006) Phobia/02 The Diary Of Jane.mp3
Breaking Benjamin/(2006) Phobia/03 Breath.mp3
Breaking Benjamin/(2006) Phobia/04 You.mp3
Breaking Benjamin/(2006) Phobia/05 Evil Angel.mp3
```

Playlist (m3u) modified to work with MyMPD & Jellyfin
```
/media/music/Apocalyptica/7th Symphony (2010)/07 - Broken Pieces.mp3
/media/music/Breaking Benjamin/(2006) Phobia/01 Intro.mp3
/media/music/Breaking Benjamin/(2006) Phobia/02 The Diary Of Jane.mp3
/media/music/Breaking Benjamin/(2006) Phobia/03 Breath.mp3
/media/music/Breaking Benjamin/(2006) Phobia/04 You.mp3
/media/music/Breaking Benjamin/(2006) Phobia/05 Evil Angel.mp3
```

**Comment by jcorporation** [comment, marked as answer]
myMPD uses the mpd protocol to generate the playlists. 

From mpd documentation:

> save_absolute_paths_in_playlists <yes or no>
> This specifies whether relative or absolute paths for song filenames are used when saving playlists. The default is “no”.

**Reply by MickMorley** [reply]
Wow, I really appreciate it! I didn't think to look in my mpd settings for something like that.
